config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties

```diff
@@ -3,6 +3,9 @@
 ## Properties files load order: component-manager/kernel -> config/default -> sakai/sakai -> sakai/local -> sakai/security
 # This file is read by the Sakai Kernel and overrides kernel.properties
 # All other sakai.properties files defined locally override these settings
+# S2U-44 Display the current user role in the portal.
+# DEFAULT: true
+# portal.display.current.role=false
```

Comment:
Move this to the portal section, after the 'portal.publish.site.confirm.enabled' property.